### DISCUSSION TITLE
Login error with "failed to find request verification token"

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#14437] [Q&A]
First time I saw that mentioned error message in any system information on support mail.
What could be possible causes for that?

**Comment by moving-bits** [comment]
`GCLogin.loginInternal` parses the retrieved page for a form field named `__RequestVerificationToken`, which contains a token that we need to transmit to the server on further calls. The error message you noticed gets thrown if we cannot find that token.
Since this seems to be a single occurrence for now, I guess it's been a broken http transmission (incomplete HTML page).
